.docker/Dockerfile

```diff
@@ -118,10 +118,14 @@ RUN sudo wget https://packages.osrfoundation.org/gazebo.gpg -O /usr/share/keyrin
 RUN sudo apt-get -q update \
     && sudo apt-get -q -y upgrade \
     && sudo apt-get -q install --no-install-recommends -y \
+    python3-pexpect \
     python3-wxgtk4.0 \
+    python3-future \
     rapidjson-dev \
     xterm \
-    libgz-sim7-dev \
+    libgz-sim?-dev \
+    gz-transport?? \
```

Comment:
These two are listed as available in the keys [here](https://github.com/osrf/osrf-rosdep/blob/master/gz/gazebo.yaml). We could probably move them to a `package.xml` (refer to below comment for more context on this - I started reviewing this from the bottom of the file)

Reply:
Makes sense.  I'm testing with the `orsf-rosdep`  files.   I wonder if those dependencies are all properly called out in `ros_gz`

Will test.